doc/content/the-things-stack/cloud/billing-and-subscription-management/_index.md

```diff
@@ -160,3 +160,9 @@ If you've reached a limit for adding devices and/or gateways, you'll be facing t
 ```
 
 If you want to add more entities than your current limit allows you to, you will need to upgrade your subscription plan. For example above, you could upgrade your plan from Discovery to Standard or Plus. If you want to upgrade make sure to check sections above, and if you face any difficulties, feel free to reach [The Things Industries sales team](mailto:sales@thethingsindustries.com).
+
+### What happens if my subscription is past due?
+
+If your {{% tts %}} tenant subscription payment is overdue, you will first receive the `Stripe subscription past due` warning when you log into your tenant's Console. If the payment is 3 weeks past due, you will encounter the `Tenant Flagged` error in the Console. This means your tenant has entered the flagged state and further functionality will be restricted because of the automatic tenant deactivation. You will probably also receive an email notifying you about overdue payment.
```

Comment:
> You will encounter the `Tenant suspended` error in the Console.
> This means your tenant has entered the suspended state.

When a tenant is flagged, a warning (the one with `Stripe subscription past due`) shows up in every action a user does, both via the Console, CLI and API.
When a tenant is suspended, the tenant is basically deactivated.

In the 3 weeks, you are first flagged, and when you still don't pay, you become suspended.